src/test/regress/input/appendonly.source

```diff
@@ -811,3 +811,19 @@ insert into fix_ao_truncate_last_sequence select 1, 1 from generate_series(1, 5)
 select count(*) from fix_ao_truncate_last_sequence;
 abort;
 
+-- check reporting of missing attributes for AO table
+create table test_reporting (id int) with (appendonly=true) distributed by (id);
```

Comment:
add drop table test. this is a standard pattern.
```sql
-- start_ignore
drop table if exists test_reporting;
drop table if exists test_reporting_r;
-- end_ignore
```

Reply:
We need to delete tables both before and after the test.
Also need to wrap the `drop table` in `-- start_ignore` and `-- end_ignore` to ignore the NOTICE if the table did not exist.

Reply:
deleted and wrapped